educative_scraper.py

```diff
@@ -122,6 +116,13 @@ def get_file_name(driver):
     return slugify(file_name[0].get_attribute('innerHTML')).replace("-", " ")
 
 
+def get_all_topic_links(driver):
+    topics_class = "CollectionSidebarLessonTitle"
+    topics = driver.find_elements(By.CLASS_NAME, topics_class)
```

Comment:
use css selector
f"a[class*='{topics_class}']"

try to keep use css selectors in selenium or js.

Reply:
This is done, thanks.
And is it recommended to always use css selectors?

Reply:
yes it helps to select the element directly in browser for debugging purposes